logger/build.gradle

```
@@ -3,7 +3,7 @@ apply plugin: 'kotlin-android'
 
 android {
   compileSdkVersion rootProject.ext.compileSdkVersion
-  buildToolsVersion rootProject.ext.buildToolsVersion
+  buildToolsVersion '28.0.3'
```

revert back to rootProject.ext.buildToolsVersion
```suggestion
  buildToolsVersion rootProject.ext.buildToolsVersion
```

and change `buildToolsVersion = '27.0.3'` to `buildToolsVersion = '28.0.3'` in logger/build.gradle instead